utils/rpms/packaging/Dockerfile.coverity

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2018-2019, Intel Corporation
+# Copyright 2018-2020, Intel Corporation
```

Comment:
Do you need to update this, and there's a permissions change on this file.

Reply:
That's an issue for daos-stack/packaging which is just being copied to here from there.